src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,23 @@
-export const UserInfo = () => {};
+interface User {
```

Comment:
Consider renaming the interface to `IUser` to follow common naming conventions. 🤓